snippets/simple-batching-update-complete.html

```diff
@@ -0,0 +1,30 @@
+<html>
+  <body>
+    <h1>Simple batching class with an <code>updateComplete</code> property</h1>
+    <h2>Open your console 😉</h2>
+    <script type="module">
+      import { Batching } from './simple-batching-update-complete.js';
+
+      /* We use an IIFE (Immediately Invoked Function Expression), because top-level await is not a thing yet 😑 */
```

Comment:
I've seen the term "Async IIFE" bandied about

also, top-level await is a thing? I think it exists in some platforms